src/telemetry/stats.h

```diff
@@ -46,13 +47,13 @@ typedef struct BaseStats
 
 typedef struct StorageStats
 {
-	BaseStats base;
+	BaseStats bs;
 	RelationSize relsize;
 } StorageStats;
 
 typedef struct HyperStats
 {
-	StorageStats storage;
+	StorageStats ss;
```

Comment:
Same applies here, it is not clear what `ss` stands for. The original name is much clearer.